### PR TITLE
Speaker tutorials

### PR DESCRIPTION
**Pull request by nithinraok**
* Added fine_tune method to set up data and modify decoder
* Added jupyter notebook for speaker recognition and verification
* Added scp_to_manifest.py script to enable users to convert any audio_directory to manifest type

### CODE REVIEW

File: examples/speaker_recognition/conf/SpeakerNet_recognition_3x2x512.yaml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  create_tensorboard_logger: True
+  create_checkpoint_callback: True
+  create_wandb_logger: False
+
+hydra:
+  run:
+    dir: .
+  job_logging:
+    root:
+      handlers: null
```
Comment (blisc) [inline examples/speaker_recognition/conf/SpeakerNet_recognition_3x2x512.yaml:172]: You can remove this part of the config

File: examples/speaker_recognition/conf/SpeakerNet_verification_3x2x512.yaml [hunk 2]
```diff
@@ -156,6 +153,7 @@ exp_manager:
   name: *name
   create_tensorboard_logger: True
   create_checkpoint_callback: True
+  create_wandb_logger: False
 
 hydra:
```
Comment (blisc) [inline examples/speaker_recognition/conf/SpeakerNet_verification_3x2x512.yaml:158]: You can remove this part of the config

File: nemo/collections/asr/models/label_models.py [hunk 3]
```diff
@@ -212,6 +215,68 @@ def multi_test_epoch_end(self, outputs, dataloader_idx: int = 0):
...
+                    new_decoder_config['params']['num_classes'], len(self.dataset.labels)
+                )
+            )
+
+        del self.decoder
+        self.decoder = EncDecSpeakerLabelModel.from_config_dict(new_decoder_config)
+
+        OmegaConf.set_struct(self._cfg.decoder, False)
+        self._cfg.decoder = new_decoder_config
+        OmegaConf.set_struct(self._cfg.decoder, True)
```
Comment (blisc) [inline nemo/collections/asr/models/label_models.py:276]: Use open_dict(), use Omry's comment here: https://github.com/NVIDIA/NeMo/pull/1049#pullrequestreview-472987867

**Review by blisc — CHANGES_REQUESTED** [review]

File: nemo/collections/asr/models/label_models.py (hunk 3 above)
Comment (nithinraok) [reply nemo/collections/asr/models/label_models.py:276]: Done, thanks

**Review by blisc — APPROVED** [review]

**Pull request merged**